ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
+  if (brand && brand.length) {
```

Comment:
We don't need to check for a string's length, we only need to do that for arrays. Empty strings in javascript are falsy.